Source/Core/Core/HW/GCMemcard/GCMemcard.h

```diff
@@ -411,9 +411,12 @@ class GCMemcard
   u16 DEntry_FirstBlock(u8 index) const;
   // get file length in blocks
   u16 DEntry_BlockCount(u8 index) const;
-  u32 DEntry_CommentsAddress(u8 index) const;
-  std::string GetSaveComment1(u8 index) const;
-  std::string GetSaveComment2(u8 index) const;
+
+  std::optional<std::vector<u8>>
+  GetSaveDataBytes(u8 save_index, size_t offset = 0,
+                   size_t length = std::numeric_limits<size_t>::max()) const;
+
+  std::optional<std::pair<std::string, std::string>> GetSaveComments(u8 index) const;
```

Comment:
This should probably have a comment indicating what each string is so others don't need to read the code to figure out its intended to be a `{title, comments}` pair.